gum/backend-posix/gumexceptor-posix.c

```diff
@@ -585,7 +585,7 @@ gum_disassemble_instruction_at (gconstpointer address,
 #if defined (HAVE_I386)
   err = cs_open (CS_ARCH_X86, GUM_CPU_MODE, &capstone);
 #elif defined (HAVE_ARM)
-  if (context->cpsr & ARM_GRP_THUMB)
+  if (context->cpsr & 0x20)
```

Comment:
```suggestion
  if ((context->cpsr & 0x20) != 0)
```

(This stylistic inconsistency went unnoticed in the PR adding the faulty code.)

Reply:
No worries. I’m just happy the feature is already in and I’ve only had to tweak one number. I noticed it was only on windows a while ago and was expecting to have to port it myself.